src/components/schedule/worker-info-section/shifts-section/shift-cell.component.tsx

```diff
@@ -196,58 +171,28 @@ export function ShiftCellComponentF(options: ShiftCellOptions): JSX.Element {
           </WrapContentDiv>
         )}
         {!showInput && (
-          <ErrorPopper
-            className="wrapContent"
-            errorSelector={errorSelector}
-            showTooltip={!showInput}
-          >
+          <ErrorWrapper errorSelector={errorSelector} showTooltip={!showInput}>
             <WrapContentDiv>
-              <div
+              <Content
                 ref={cellRef}
-                className={`content ${hasNextClass} ${
-                  keepOn ? "keepOnTrue" : "keepOnFalse"
-                } ${keepOnClass}`}
                 style={cellStyle}
                 data-cy={baseCellDataCy(cellIndex, "highlighted-cell")}
               >
                 {!keepOn && !SHIFTS[shiftCode].isWorkingShift && shiftCode !== "W" && (
-                  <div
-                    style={{ backgroundColor: `#${getColor(shiftCode, shiftTypes)}` }}
-                    className={"leftBorder"}
-                  />
+                  <ShiftBar style={{ backgroundColor: `#${getColor(shiftCode, shiftTypes)}` }} />
                 )}
-                <p
+                <Shift
                   data-cy={baseCellDataCy(cellIndex, "cell")}
-                  className={"relative "}
                   style={{ color: `#${getColor(shiftCode, shiftTypes)}` }}
                 >
                   {keepOn || shiftCode === ShiftCode.W ? "" : shiftCode}
-                </p>
-              </div>
+                </Shift>
+              </Content>
             </WrapContentDiv>
-          </ErrorPopper>
+          </ErrorWrapper>
         )}
-        {/* TODO revert cell details
-         {!isEditMode && (
-           <Popper
-             ref={cellDetailsPopperRef}
-             className="cell-details-poppers"
-             style={styles}
-             isOpen={isComponentVisible && isBlocked && value !== ""}
-           >
-             <CellDetails
-               index={cellIndex}
-               day={verboseDate?.date ?? 0}
-               month={monthNumber ?? new Date().getMonth()}
-               year={year}
-               shiftcode={value}
-               {...options}
-               close={(): void => setIsComponentVisible(false)}
-             />
-           </Popper>
-         )} */}
```

Comment:
Umyślnie to zostawiłem żeby potem łatwiej było to przywrócić. 
Był z tym jakiś problem?